src/background/services/monetization.ts

```diff
@@ -26,8 +26,8 @@ export class MonetizationService {
     payload: StartMonetizationPayload,
     sender: Runtime.MessageSender
   ) {
-      // TODO: This is not ideal. We should not receive monetization events
-      // from the content script if WM is disabled or a wallet is not connected.
+    // TODO: This is not ideal. We should not receive monetization events
+    // from the content script if WM is disabled or a wallet is not connected.
     const { connected, enabled } = await this.storage.get([
       'enabled',
       'connected'
```

Comment:
not related to your changes, but should this be 
```suggestion
    const { connected, enabled } = await this.storage.get([
      'connected',
      'enabled'
```
instead?

Reply:
not necessary, the get method accepts an array of keys and returns an object. There is destructuring of the object so we don't care about the order.